Lib/test/test_tarfile.py

```diff
@@ -568,6 +568,21 @@ def test_int_name_attribute(self):
             with tarfile.open(fileobj=fobj, mode=self.mode) as tar:
                 self.assertIsNone(tar.name)
 
+    def _test_name(self, tarname):
```

Comment:
```suggestion
    def test_name(self, tarname):
```
is underscore here intentional?

Reply:
No, it would be incorrect, since this method has a required parameter. It should be simply deleted. It is a remnant of my attempt to generalize `test_pathlike_name` for other types, but I decided that for bytes path the existing test is good enough. Thank you for noticing this.